src/AzureStack/SubscriptionsAdmin/Subscriptions.Admin.Tests/src/AcquiredPlanTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+
+using System.Linq;
+
+using Microsoft.AzureStack.Management.Subscriptions.Admin;
+using Microsoft.AzureStack.Management.Subscriptions.Admin.Models;
+using System;
+using Xunit;
+
+namespace Subscriptions.Tests
+{
+    public class AcquiredPlanTests : SubscriptionsTestBase
+    {
+
+        private void ValidatePlanAcquisition(PlanAcquisition planAcquisition) {
+            // Resource
```

Comment:
We recommend skipping these tests instead of commenting them out
Applies to all tests here